crates/stages/src/stages/bodies.rs

```diff
@@ -628,11 +629,9 @@ mod tests {
                             tx.put::<tables::TxTransitionIndex>(tx_id, tx_id)
                         })?;
 
-                        // Randomize rewards
-                        let has_reward: bool = rand::random();
-                        let last_transition_id = progress.body.len().saturating_sub(1) as u64;
-                        let block_transition_id =
-                            last_transition_id + if has_reward { 1 } else { 0 };
+                        let last_transition_id = progress.body.len() as u64;
+                        let block_transition_id = last_transition_id + 1; // for block reward
+                        println!("Bodies tx:{}", progress.body.len());
```

Comment:
Why do we no longer randomly check if there's a block reward? Or doesn't matter is the point?